hack/test-example.sh

```diff
@@ -78,6 +82,14 @@ export ftp_proxy=http://localhost:2121
 
 INFO "Starting \"$NAME\" from \"$FILE\""
 defer "limactl delete -f \"$NAME\""
+
+if [[ -n ${CHECKS["disk"]} ]]; then
+	if ! limactl disk ls | grep -q "^data\s"; then
```

Comment:
This is an example where it would be useful to be able to pass disk names to the `ls` command:

```
if ! limactl disk ls data; then
   ...
fi
```